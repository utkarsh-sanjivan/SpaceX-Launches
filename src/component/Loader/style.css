.spacex-loader {
    color: #c5df9d;
    font-size: 200px;
    margin: 25% 0% 0% 25%;
}

@media only screen and (max-width: 700px) {
    .spacex-loader {
        margin: 0;
    }
}