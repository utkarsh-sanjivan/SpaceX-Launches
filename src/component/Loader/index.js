import React from 'react';
import './style.css';

const Loader = () => (
    <i class="fas fa-spin fa-cog spacex-loader"></i>
);

export default Loader;