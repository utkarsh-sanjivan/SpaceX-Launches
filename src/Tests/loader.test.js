import React from 'react';
import { create } from "react-test-renderer";
import Loader from './../component/Loader/index';

test('Test SpaceX Loader Component', () => {
  const component = create(<Loader />);
  expect(component.toJSON()).toMatchSnapshot()
});