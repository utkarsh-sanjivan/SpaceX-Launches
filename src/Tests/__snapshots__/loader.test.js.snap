// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SpaceX Loader Component 1`] = `
<i
  class="fas fa-spin fa-cog spacex-loader"
/>
`;
